.sequelizerc

```diff
@@ -0,0 +1,7 @@
+require('@babel/register');
```

Comment:
I think this can be changed to ES6. you can use `import` instead

Reply:
I tried and it didn't work, but I think it's because .sequelizerc is a setup file for sequelize.

Is there a solution you can suggest?